frontend/benefit/handler/src/components/applicationsArchive/ApplicationsArchive.tsx

```diff
@@ -41,28 +84,88 @@ const ApplicationsArchive: React.FC = () => {
       <>
         <div style={{ maxWidth: 630 }}>
           <SearchInput
-            helperText={t('common:search.input.keyword.helperText')}
-            label={t('common:search.input.keyword.label')}
-            placeholder={t('common:search.input.keyword.placeholder')}
+            helperText={t(
+              'common:search.fields.searchInput.keyword.helperText'
+            )}
+            label={t('common:search.fields.searchInput.keyword.label')}
+            placeholder={t(
+              'common:search.fields.searchInput.keyword.placeholder'
+            )}
             onChange={(e) => setSearchString(e)}
             onSubmit={(e) => onSearch(e)}
             css="margin-bottom: var(--spacing-m);"
           />
         </div>
 
-        <ApplicationArchiveList
-          data={searchResults?.matches}
-          searchString={searchString}
-          submittedSearchString={submittedSearchString}
-          isSearchLoading={isSearchLoading}
-        />
+        <$Grid>
+          <$GridCell $colSpan={12}>
+            <SelectionGroup>
+              <Checkbox
+                id="include-archival-applications"
+                label="Näytä hyväksytyt vanhat hakemukset"
```

Comment:
This isn't added to translations.

Reply:
Will be removed later on, sticking to this now.